overrides/app/components/algolia/autocomplete/components/Breadcrumb.tsx

```diff
@@ -1,7 +1,7 @@
-import React from 'react';
-import { intersperse } from '../utils';
+import React from 'react'
+import {intersperse} from '../utils'
```

Comment:
Did you apply some prettifier? The previous indentation looks better (with the spaces) and is the one we have in our doc and examples
It also adds noise to an already big review. Could you revert?

Reply:
Hey @sbellone ,
I actually changed these styles to make it suitable with PWA-Kit general code style. I can sure try to revert it and create a separate PR but honestly, I also made an upgrade from `react-instantsearch-hooks-web` library to `react-instantsearch` and have concern to break sth. Sorry, to this mess up

Reply:
Ok I see that it's indeed the default PWA style. I understand :+1: